example/android/app/src/main/kotlin/rekab/app/example/MainActivity.kt

```diff
@@ -0,0 +1,6 @@
+package rekab.app.example
```

Comment:
I believe this file is useless. please remove it;

Reply:
removed in latest commit